CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.19.8
```

Comment:
Please bump the version to `0.20.0` instead of just a patch release. This update includes a breaking change to the SDK version (from 18 → 19), which may affect dependent apps. A minor version bump signals this change clearly and avoids accidental upgrades.